packages/block-editor/src/components/url-input/index.js

```diff
@@ -453,7 +455,6 @@ class URLInput extends Component {
 			placeholder,
 			onKeyDown: this.onKeyDown,
 			role: 'combobox',
-			'aria-label': __( 'URL' ),
```

Comment:
Instead of removing aria-label here, would it be better to do this?

```
aria-label={ label ? undefined : __( 'URL' ) }
```

The undefined will strip the attribute on the front-end and allow us to not add more lines for simple conditional. Just a thought.

Reply:
Thanks, @alexstine. I like the change, especially since it doesn't make the intent less clear and is more concise (double win). I added an inline comment as well to provide more context to the decision.